packages/plugins/response-cache/src/plugin.ts

```diff
@@ -191,15 +198,10 @@ export function useResponseCache({
       };
       replaceSchema(patchedSchema);
     },
-    onParse(parseCtx) {
-      return function onParseEnd(ctx) {
-        if (ctx.result && 'kind' in ctx.result) {
-          const source = parseCtx.params.source;
-
-          const rawDocumentString = typeof source === 'string' ? source : source.body;
-          ctx.extendContext({
-            [rawDocumentStringSymbol]: rawDocumentString,
-          });
+    onParse({ params: { source } }) {
```

Comment:
Nit: In case a custom `getDocumentString` function is set we can omit/skip this `onParse` hook completly?

Reply:
Technically yes. Hmm so only if default document string getter is used, onParse is needed. I can refactor there.

Reply:
@ardatan thank you